packages/swapper/src/swappers/JupiterSwapper/utils/helpers.ts

```diff
@@ -0,0 +1,72 @@
+import { type ChainId, fromAssetId } from '@shapeshiftoss/caip'
+import type { Result } from '@sniptt/monads'
+import type { AxiosResponse } from 'axios'
+
+import type { SwapErrorRight } from '../../../types'
+import type { QuoteResponse } from '../models/QuoteResponse'
+import type { SwapInstructionsResponse } from '../models/SwapInstructionsResponse'
+import { jupiterSupportedChainIds } from './constants'
+import { jupiterService } from './jupiterService'
+
+export const isSupportedChainId = (chainId: ChainId): chainId is ChainId => {
+  return jupiterSupportedChainIds.includes(chainId)
+}
+
+const JUPITER_TRANSACTION_MAX_ACCOUNTS = 54
+
+type GetJupiterQuoteArgs = {
+  apiUrl: string
+  sourceAsset: string
+  destinationAsset: string
+  commissionBps: string
+  amount: string
+  slippageBps?: string
```

Comment:
nitpick: make this required as it will actually always be defined currently:

https://github.com/shapeshift/web/blob/f7317eeb8a0b22774a66a2bc84eb0f6a7dfcabeb/packages/swapper/src/swappers/JupiterSwapper/swapperApi/getTradeQuote.ts#L86-L89

But if it wasn't, we wouldn't catch it

Reply:
Made in [4e76e81](https://github.com/shapeshift/web/pull/8120/commits/4e76e8115223d91bd602f7d9ceed80c94c77ea15)